Documentation/gettingstarted_kafka.rst

```diff
@@ -0,0 +1,392 @@
+For starters, the outpost container can actually switch roles from a consumer to a producer,
+sending "malicious" data to all other consumers on the topic.
+
+To prove this, kill the existing kafka-consume.sh command in the outpost-9999 window
+by typing control-C and instead run:
+
+::
+
+  $ echo "Vader Booed at Empire Karaoke Party" | ./kafka-produce.sh --topic empire-announce
+
+Uh oh!  Outpost-8888 and all of the other outposts in the empire have now received this fake announcement.
+
+But even more nasty from a security perspective is that the outpost container can access any topic
+on the kafka-broker.
+
+In the outpost-9999 container, run:
+
+::
+
+  $ ./kafka-consume.sh --topic deathstar-plans
+  "deathstar reactor design v3"
+
+We see that any outpost can actually access the secret deathstar plans.  Now we now how the rebels got
+access to them!
+
+Step 6: Securing Access to Kafka with Cilium
+============================================
+
+Obviously, it would be much more secure to limit each pod's access to the Kafka broker to be
+least privilege (i.e., only what is needed for the app to operate correctly and nothing more).
+
+We can do that with the following Cilium security policy.   As with Cilium HTTP policies, we can write
+policies that identify pods by labels, and then limit the traffic in/out of this pod.  In
+this case, we'll create a policy that identifies exactly the traffic that should be allowed to reach the
+Kafka broker, and deny the rest.
+
+As an example, a policy could limit containers with label *app=empire-outpost* to only be able to consume
+topic *empire-announce*, but would block any attempt by a compromised container (e.g., empire-outpost-9999)
+from producing to *empire-announce* or consuming from *deathstar-plans*.
+
+.. image:: images/cilium_kafka_gsg_attack.png
+
+Here is the CiliumNetworkPolicy rule that limits access of pods with label *app=empire-outpost* to
+only consume on topic *empire-announce*:
+
+::
+
+  apiVersion: "cilium.io/v2"
+  kind: CiliumNetworkPolicy
+  description: "enable outposts to consume empire-announce"
+  metadata:
+    name: "rule2"
+  spec:
+    endpointSelector:
+      matchLabels:
+        app: kafka
+    ingress:
+    - fromEndpoints:
+      - matchLabels:
+          app: empire-outpost
+      toPorts:
+      - ports:
+        - port: "9092"
+          protocol: TCP
+        rules:
+          kafka:
+          - apiKey: "fetch"
+            topic: "empire-announce"
+          - apiKey: "apiversions"
+          - apiKey: "metadata"
+          - apiKey: "findcoordinator"
+          - apiKey: "joingroup"
+          - apiKey: "leavegroup"
+          - apiKey: "syncgroup"
+          - apiKey: "offsets"
+          - apiKey: "offsetcommit"
+          - apiKey: "offsetfetch"
+          - apiKey: "heartbeat"
+
+
+CiliumNetworkPolicies operate by white-listing allowed traffic, meaning that requests that do not match
```

Comment:
Suggest to italicize, or break up into separate words for "CiliumNetworkPolicies".